src/options.cpp

```diff
@@ -972,7 +972,7 @@ Options::Options()
     this->Register(&m_bracketThickness, "bracketThickness", &m_generalLayout);
 
     m_dynamDist.SetInfo("Dynam dist", "The default distance from the staff for dynamic marks");
-    m_dynamDist.Init(1.0, 0.5, 8.0);
+    m_dynamDist.Init(4.0, 1.0, 16.0);
```

Comment:
Don't change the default without any documented reason

Reply:
I guess before it was just applied to the top/bottom margin.

Reply:
Maybe. But I can see this, which is not statisfactory

<img width="674" alt="image" src="https://user-images.githubusercontent.com/689412/114666303-61b7e500-9cfe-11eb-85c6-07ceadc25497.png">

Reply:
What is actually not satisfactory here is my encoding, as the dim. should be connected to the upper staff. If this is corrected the outcome is this: 
![layer_012](https://user-images.githubusercontent.com/7693447/114667487-d0497280-9cff-11eb-8620-5407d0759c3b.png)

Reply:
But keeping the old default is also ok.